content/en/docs/distributions/gke/deploy/delete-cli.md

```diff
@@ -33,7 +33,7 @@ This guide assumes the following settings:
 
     ```bash
     cd "${KF_DIR}"
-    make delete
+    make delete-gcp
```

Comment:
```suggestion
    make delete
```

Reply:
Thanks, I didn't realize it was changed recently.